pkg/menu/menu_test.go

```diff
@@ -0,0 +1,105 @@
+package menu
+
+import (
+	ui "github.com/gizak/termui/v3"
+	"strconv"
+	"testing"
+)
+
+func Test_newParagraph(t *testing.T) {
```

Comment:
Nit: I think convention in Go is to not use underscores for naming so you can name your tests TestXxx, so this would become `TestNewParagraph`